### PR TITLE
Update celo-dapp-gallery.md

### PR DESCRIPTION
**Pull request by thelazyceo1**
## Description

_A few sentences describing the overall effects and goals of the pull request's commits.
What is the current behavior, and what is the updated/expected behavior with this PR?_

## Other changes

_Describe any minor or "drive-by" changes here._

## Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

## Related issues

- Fixes #[issue number here]

## Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

## Commit Message
_If using automerge, specify desired commit title_

_And desired commit message body_

### CODE REVIEW

File: packages/docs/developer-resources/celo-dapp-gallery.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ Savings Circles let you pool funds with your friends to save for large purchases
 
 Github: https://github.com/celo-org/savings-circle-demo
 
+## Dunia Pay Wallet 
+
+At Dunia Payment , we are building an electronic wallet that will let poeple in Sub Saharan Africa send and receive money directly on their phone.The app will use the celo light client to process transaciton faster even in low internet connexion area common in Africa.The wallet will be built on top of the celo platform and also using a set of external open source smart contracts.
```
Comment (critesjosh) [inline packages/docs/developer-resources/celo-dapp-gallery.md]: A few minor English edits

```suggestion
At Dunia Payment, we are building an electronic wallet that will let people in Sub Saharan Africa send and receive money directly on their phone.The app will use the Celo light client to process transactions faster, even in low internet connection areas common in Africa. The wallet will be built on top of the Celo platform and also using a set of external open source smart contracts.
```

Comment (critesjosh) [reply packages/docs/developer-resources/celo-dapp-gallery.md]: if you could update, then I'll approve

**Review by critesjosh — COMMENTED** [review]
if you could update the proposed changes, then I'll approve

File: packages/docs/developer-resources/celo-dapp-gallery.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ Savings Circles let you pool funds with your friends to save for large purchases
 
 Github: https://github.com/celo-org/savings-circle-demo
 
+## Dunia Pay Wallet 
+
+At Dunia Payment, we are building an electronic wallet that will let people in Sub Saharan Africa send and receive money directly on their phone.The app will use the Celo light client to process transactions faster, even in low internet connection areas common in Africa. The wallet will be built on top of the Celo platform and also using a set of external open source smart contracts.
```
Comment (critesjosh) [inline packages/docs/developer-resources/celo-dapp-gallery.md:55]: Minor formatting nit pick

```suggestion
At Dunia Payment, we are building an electronic wallet that will let people in Sub Saharan Africa send and receive money directly on their phone. The app will use the Celo light client to process transactions faster, even in low internet connection areas common in Africa. The wallet will be built on top of the Celo platform and also using a set of external open source smart contracts.
```

**Review by thelazyceo1 — COMMENTED** [review]
Done!😁

**Review by critesjosh — APPROVED** [review]

**Pull request merged**